src/util/glibc_workaround.cpp

```diff
@@ -7,13 +7,17 @@
 // This is needed because libstdc++ itself uses this API - its not
 // just an issue of your code using it, ughhh
 
+#pragma GCC diagnostic push
```

Comment:
We can probably remove this whole file. 
We're not supporting GCC versions this old anymore.